atom/renderer/api/atom_api_web_frame.cc

```diff
@@ -368,6 +368,116 @@ void WebFrame::ClearCache(v8::Isolate* isolate) {
     base::MemoryPressureListener::MEMORY_PRESSURE_LEVEL_CRITICAL);
 }
 
+v8::Local<v8::Value> WebFrame::Opener() const {
+  blink::WebLocalFrame* local_frame =
+    (blink::WebLocalFrame*) WebFrame::web_frame_->Opener();
```

Comment:
Its possible for a `blink::WebFrame` to be either `blink::WebLocalFrame` or `blink::WebRemoteFrame` when traversing frames across process, this model isn't currently enabled in Electron but will soon be. So its better to prepare for it by using the helpers `blink::WebFrame::IsWebLocalFrame` and `blink::WebFrame::ToWebLocalFrame`